package.json

```diff
@@ -26,10 +26,10 @@
   "dependencies": {
     "front-matter": "^3.0.0",
     "loader-utils": "^1.1.0",
-    "markdown-it": "^8.4.2"
+    "markdown-it": "^8.4.2",
+    "vue-template-es2015-compiler": "^1.6.0"
```

Comment:
I avoided having in `dependencies` for the user who is not interested in Vue. That's why I've put to `optionalDependencies`. Could you remove that, and I will leave about that on README around where is mentioning Vue.